README.md

```diff
@@ -105,14 +105,17 @@ Located in the [app](app) directory:
 
 
 ### To test locally
+The app can be tested and developed locally with a deployed app in place. The reason is that access is guarded by an access token. That token can only be issued in the Kubernetes environment with App ID intercepting requests.
```

Comment:
I had to read this section a few times. Suggest the following to make it easier when I go and read again in a few months. 
> The app can be tested and developed locally, however it requires a version (same or different to the local version) of the app to be deployed in Kubernetes. The reason is ...